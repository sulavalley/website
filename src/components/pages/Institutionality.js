import React from 'react';

const Institutionality = () => {
  return (
    <div>
      <h1>Estas en Institucionalidad</h1>
    </div>
  );
};

export default Institutionality;
