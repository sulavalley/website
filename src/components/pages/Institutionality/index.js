import React from 'react';
import { Item, Header } from 'semantic-ui-react';
import image3 from '../../../assets/images/image3.jpg';
import image5 from '../../../assets/images/image5.jpg';
import image22 from '../../../assets/images/image22.jpg';
import image28 from '../../../assets/images/image28.jpg';

const loremIpsum =
  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';

const Institutionality = () => {
  return (
    <Item.Group divided>
      <Item>
        <Item.Image size="medium" src={image3} />
        <Item.Content>
          <Header size="huge">Misión</Header>
          <Item.Description>
            <p>{loremIpsum}</p>
          </Item.Description>
        </Item.Content>
      </Item>

      <Item>
        <Item.Image size="medium" src={image5} />
        <Item.Content>
          <Header size="huge">Visión</Header>
          <Item.Description>
            <p>{loremIpsum}</p>
          </Item.Description>
        </Item.Content>
      </Item>

      <Item>
        <Item.Image size="medium" src={image22} />
        <Item.Content>
          <Header size="huge">Objetivos</Header>
          <Item.Description>
            <p>{loremIpsum}</p>
          </Item.Description>
        </Item.Content>
      </Item>

      <Item>
        <Item.Image size="medium" src={image28} />
        <Item.Content>
          <Header size="huge">Alcance</Header>
          <Item.Description>
            <p>{loremIpsum}</p>
          </Item.Description>
        </Item.Content>
      </Item>
    </Item.Group>
  );
};

export default Institutionality;
