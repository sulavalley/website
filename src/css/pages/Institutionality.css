img.grid-image {
  max-width: 128px;
  max-height: 198px;
  min-width: 128px;
  min-height: 198px;
}

@media only screen and (max-width: 990px) {
  .image-container {
    display: none;
  }
}
