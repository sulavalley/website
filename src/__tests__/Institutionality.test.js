import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Institutionality from './../components/pages/Institutionality';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Institutionality />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<Institutionality />);
});

test('renders correctly', () => {
  const tree = renderer.create(<Institutionality />).toJSON();
  expect(tree).toMatchSnapshot();
});
