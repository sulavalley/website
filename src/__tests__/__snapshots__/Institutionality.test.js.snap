// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui items"
>
  <div
    align="center"
    className="item"
  >
    <div
      className="content"
    >
      <h1>
        Misión
      </h1>
      <div
        className="description"
        style={
          Object {
            "maxWidth": "50%",
          }
        }
      >
        <p>
          Nulla aliquet elit non nulla pellentesque placerat sed malesuada metus. Fusce ac odio et augue convallis consequat. Fusce ullamcorper nulla vitae dui molestie tristique cursus suscipit massa. Suspendisse pulvinar quam sed orci bibendum vestibulum. Curabitur libero neque, interdum ornare mi et, auctor hendrerit lacus. Aliquam erat volutpat. In sollicitudin convallis hendrerit. In vel sodales diam, vitae sagittis massa. Nunc vulputate faucibus felis, efficitur semper turpis imperdiet quis.
        </p>
      </div>
    </div>
    <div
      className="image"
    >
      <img
        className="ui image"
        src="image3.jpg"
      />
    </div>
  </div>
  <div
    className="item"
  >
    <div
      className="image"
    >
      <img
        className="ui image"
        src="image5.jpg"
      />
    </div>
    <div
      align="center"
      className="content"
    >
      <h1>
        Visión
      </h1>
      <div
        className="description"
        style={
          Object {
            "maxWidth": "50%",
          }
        }
      >
        <p>
          Nulla aliquet elit non nulla pellentesque placerat sed malesuada metus. Fusce ac odio et augue convallis consequat. Fusce ullamcorper nulla vitae dui molestie tristique cursus suscipit massa. Suspendisse pulvinar quam sed orci bibendum vestibulum. Curabitur libero neque, interdum ornare mi et, auctor hendrerit lacus. Aliquam erat volutpat. In sollicitudin convallis hendrerit. In vel sodales diam, vitae sagittis massa. Nunc vulputate faucibus felis, efficitur semper turpis imperdiet quis.
        </p>
      </div>
    </div>
  </div>
  <div
    className="item"
  >
    <div
      align="center"
      className="content"
    >
      <h1>
        Objetivos
      </h1>
      <div
        className="description"
        style={
          Object {
            "maxWidth": "50%",
          }
        }
      >
        <p>
          Nulla aliquet elit non nulla pellentesque placerat sed malesuada metus. Fusce ac odio et augue convallis consequat. Fusce ullamcorper nulla vitae dui molestie tristique cursus suscipit massa. Suspendisse pulvinar quam sed orci bibendum vestibulum. Curabitur libero neque, interdum ornare mi et, auctor hendrerit lacus. Aliquam erat volutpat. In sollicitudin convallis hendrerit. In vel sodales diam, vitae sagittis massa. Nunc vulputate faucibus felis, efficitur semper turpis imperdiet quis.
        </p>
      </div>
    </div>
    <div
      className="image"
    >
      <img
        className="ui image"
        src="image22.jpg"
      />
    </div>
  </div>
  <div
    className="item"
  >
    <div
      className="image"
    >
      <img
        className="ui image"
        src="image28.jpg"
      />
    </div>
    <div
      align="center"
      className="content"
    >
      <h1>
        Alcance
      </h1>
      <div
        className="description"
        style={
          Object {
            "maxWidth": "50%",
          }
        }
      >
        <p>
          Nulla aliquet elit non nulla pellentesque placerat sed malesuada metus. Fusce ac odio et augue convallis consequat. Fusce ullamcorper nulla vitae dui molestie tristique cursus suscipit massa. Suspendisse pulvinar quam sed orci bibendum vestibulum. Curabitur libero neque, interdum ornare mi et, auctor hendrerit lacus. Aliquam erat volutpat. In sollicitudin convallis hendrerit. In vel sodales diam, vitae sagittis massa. Nunc vulputate faucibus felis, efficitur semper turpis imperdiet quis.
        </p>
      </div>
    </div>
  </div>
</div>
`;
